deepspeed/runtime/domino/transformer.py

```diff
@@ -356,20 +382,73 @@ def forward(self, hidden_states, attention_mask, rotary_pos_emb=None):
         if not self.llama_model:
             rotary_pos_emb = None
 
-        attention_output0, attention_bias0  = \
-            self.self_attention(
-                layernorm_output0,
+        if self.input_split_dim == "seq":
+            # Compute full Q, K, V
+            layernorm_output = torch.concat([layernorm_output0, layernorm_output1], dim=0)
```

Comment:
this is wrong. our original layernorm_output0, layernorm_output1 are split on batch as `dim=1`,  concat on `dim=0` is incorrect.